packages/core/src/components/forms/_label.scss

```diff
@@ -31,6 +31,10 @@ label.#{$ns}-label {
   display: block;
   margin: 0 0 ($pt-grid-size * 1.5);
 
+  strong {
```

Comment:
this does not belong here. 1) it should already be true. 2) if not, it should be applied such that all `strong` tags look the same in blueprint.

Reply:
1) not true yet 2) not that simple. the boldness is a function of the font size.

since 600 is the most common "bold" weight for 14px text (our default UI font size), I suppose it's fair to default `<strong>` to 600 weight. I can then remove this rule, 👍 or 👎 ?

Reply:
sounds good, I'll address as soon as I can!

Reply:
Done!